api.d.ts

```diff
@@ -186,6 +186,53 @@ export interface BundlingAPI {
 
 export declare const bundling: BundlingAPI
 
+// ----------------------------------------------------------------
+// -------------------------- SDK ----------------------------
+// ----------------------------------------------------------------
+
+export interface SdkAPI {
+  /**
+   * create SDK json files
+   */
+  generateSdkDocumentation: (
+    target: string,
```

Comment:
Maybe a comment that target is the path to the package.json of the extension/feature?